Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 # StackStorm version which refers to Docker images tag
-appVersion: 3.4dev
+appVersion: "3.4"
```

Comment:
Please keep the version as before `3.4dev`. I'll bump to the latest one in another PR.